superset-frontend/src/explore/components/controls/DateFilterControl/DateFilterLabel.tsx

```diff
@@ -174,11 +172,6 @@ export default function DateFilterLabel(props: DateFilterControlProps) {
           guessedFrame === 'No filter'
         ) {
           setActualTimeRange(value);
-          setTooltipTitle(
-            type === ('error' as Type)
-              ? t('Default value is required')
-              : actualRange || '',
-          );
```

Comment:
Since the default value always gets from `Store` or `constants`, the value of the `DateFilterControl` will not be undefined.

So the type in the `DateFilterControl` is useless.